Formula/zsh-you-should-use.rb

```diff
@@ -0,0 +1,37 @@
+class ZshYouShouldUse < Formula
+  desc "ZSH plugin that reminds you to use existing aliases for commands you just typed"
+  homepage "https://github.com/MichaelAquilina/zsh-you-should-use"
+  url "https://github.com/MichaelAquilina/zsh-you-should-use/archive/1.7.3.tar.gz"
+  sha256 "db4486cd12974332ec858d446aff9393dae6be430d425a56d7036d2ce4edeb9e"
+  license "GPL-3.0-or-later"
+
+  bottle :unneeded
+
+  depends_on "zsh" => "5.1"
```

Comment:
You shouldn't specify versions here. If it doesn't work with the latest version it shouldn't be in homebrew.

Reply:
I thought that's how you wrote minimum version, but I have removed it.